lib/chai/utils/inspect.js

```diff
@@ -37,6 +37,7 @@ var isDOMElement = function (object) {
   } else {
     return object &&
       typeof object === 'object' &&
+      Object.hasOwnProperty.call(object, 'nodeType') &&
```

Comment:
I wonder if it'd be safer to do a `typeof` on line 40, just in the off chance that an object is inheriting `nodeType` from its prototype chain.